genai-perf/README.md

```diff
@@ -407,9 +400,9 @@ URL of the endpoint to target for benchmarking. (default: `None`)
 ##### `--batch-size <int>`
 ##### `--batch-size-text <int>`
 
-The text batch size of the requests GenAI-Perf should send.
+The batch size of the requests GenAI-Perf should send.
```

Comment:
This looks like it was lot during rebase (thanks for rebasing!). Could you return the word "text" here to differentiate from --batch-size-image?

Reply:
Thank you for catching this. Done